client/src/common/components/Screen/Screen.library.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {ScrollView} from 'react-native-gesture-handler';
+import {Display36} from '../Typography/Display/Display';
+import Screen from './Screen';
+
+export const DefaultScreen = () => (
+  <Screen>
+    <ScrollView>
+      <Display36>
+        DefaultScreen. Lorem Ipsum is simply dummy text of the printing and
+        typesetting industry. Lorem Ipsum has been the industry's standard dummy
+        text ever since the 1500s.
+      </Display36>
+    </ScrollView>
+  </Screen>
+);
+export const DefaultWithBackgroundColor = () => (
+  <Screen backgroundColor="red">
+    <Display36>
+      WithBackgroundColor. Lorem Ipsum is simply dummy text of the printing and
+      typesetting industry. Lorem Ipsum has been the industry's standard dummy
+      text ever since the 1500s.
+    </Display36>
+  </Screen>
+);
+export const DefaultWithTextColor = () => (
```

Comment:
This one doesn't really make sense as there is no visual change in the UI library. How about having a backbutton here too?

Reply:
Hahahaha 🤦‍♀️